packages/next-templates/src/app/wmebv/productenpagina-klachtenformulier/page.tsx

```diff
@@ -0,0 +1,103 @@
+'use client';
+
+import {
+  UtrechtArticle,
+  UtrechtButton,
+  UtrechtButtonGroup,
+  UtrechtDigidLogo,
+  UtrechtHeading1,
+  UtrechtHeading2,
+  UtrechtIcon,
+  UtrechtLink,
+  UtrechtPage,
+  UtrechtPageContent,
+  UtrechtParagraph,
+} from '@utrecht/web-component-library-react';
+import { ExampleHeaderFunnelWmebv } from '@/components/wmebv/Header/ExampleHeaderFunnelWmebv';
+import { ExampleFooterWmebv } from '@/components/wmebv/Footer/ExampleFooterWmebv';
+import { ExampleNavigation } from '@/components/ExampleNavigation/ExampleNavigation';
```

Comment:
Hier maak je geen gebruik van, dan kan het weg

Reply:
Ook dit is van een oudere commit, in de huidge versie zijn deze eruit! 😄